expert/ezproxy.cfg

```diff
@@ -112,6 +112,23 @@ IncludeFile databases/jstor.txt
 # https://help.oclc.org/Library_Management/EZproxy/Database_stanzas/Nomos_eLibrary
 IncludeFile databases/nomoselibrary.txt
 
+# OECD iLibrary
+# https://help.oclc.org/Library_Management/EZproxy/Database_stanzas/OECD_iLibrary?sl=en
+Title OECD iLibrary
+IncludeFile databases/oecdilibrary.txt
```

Comment:
Looks like the neverproxy in this config is Preventing OECD data from working.
We've encountered this before, and the answer is to copy over the hosted config and remove the offending line, eg:
```suggestion
# Modified from
# https://help.oclc.org/Library_Management/EZproxy/Database_stanzas/OECD_iLibrary?sl=en
# to allow OECD Data to function
Title OECD iLibrary (updated 20180530)
HTTPHeader -request -process X-Requested-With
URL https://www.oecd-ilibrary.org/
DJ oecd.org
DJ oecdilibrary.org
DJ oecd-ilibrary.org
DJ nordic-ilibrary.org
DJ thecommonwealth-ilibrary.org
DJ un-ilibrary.org
DJ itu-ilibrary.org
DJ wto-ilibrary.org
HJ www.oecd.org
HJ www.oecdilibrary.org
HJ www.oecd-ilibrary.org
HJ www.nordic-ilibrary.org
HJ www.thecommonwealth-ilibrary.org
HJ www.un-ilibrary.org
HJ www.itu-ilibrary.org
HJ www.wto-ilibrary.org
HJ read.oecd-ilibrary.org
HJ www.keepeek.com
HJ https://www.oecd.org
HJ https://www.oecdilibrary.org
HJ https://www.oecd-ilibrary.org
HJ https://www.nordic-ilibrary.org
HJ https://www.thecommonwealth-ilibrary.org
HJ https://www.un-ilibrary.org
HJ https://www.itu-ilibrary.org
HJ https://www.wto-ilibrary.org
HJ https://read.oecd-ilibrary.org
HJ https://www.keepeek.com
Find xmlns=\"http://
Replace xmlns=\"http://^A
Find LENS_BASE_URI = "http://
Replace LENS_BASE_URI = "http://^A
Find CONTENT_BASE_URI = "http://
Replace CONTENT_BASE_URI = "http://^A
Find param="http://
Replace param="http://^A
Find xmlns=\"https://
Replace xmlns=\"https://^A
Find LENS_BASE_URI = "https://
Replace LENS_BASE_URI = "https://^A
Find CONTENT_BASE_URI = "https://
Replace CONTENT_BASE_URI = "https://^A
Find param="https://
Replace param="https://^A
NeverProxy oecd.metastore.ingenta.com
NeverProxy stats.oecd.org
```